packages/e2e-tests/next-app/cypress/integration/pages.test.ts

```diff
@@ -70,6 +70,22 @@ describe("Pages Tests", () => {
         cy.visit(path);
       });
 
+      it(`supports preview mode ${path}`, () => {
+        cy.request("/api/preview/enabled");
+        cy.getCookies().should("have.length", 2); // Preview cookies are now set
+        // FIXME: Not sure why adding cy.ensureRouteNotCached(path) here fails as a preview response should be uncached?
```

Comment:
@dphang I was wondering if this is entirely because we are using a static page for these tests. I think adding a server rendered example might clear this up. The `/ssg-page` should just be built as `ssg-page.html` by next as it does not contain a `getStaticPaths` therefore it is just a single HTML page. WDYT?

Reply:
Yeah it would be a static page, but I think when you pass cookies it actually becomes a cache miss to CloudFront and preview mode will tell it to always do an SSR, not use the page from S3.

This seems to be the case when I tested locally, but just not sure why cypress was still thinking it's cached somehow?